packages/gafl-webapp-service/src/pages/layout/layout.njk

```diff
@@ -170,7 +175,13 @@
    {% endif %}
 {% endblock %}
 
-{% block bodyStart %}{% endblock %}
+{% block bodyStart %}
+{% if gtmContainerId %}
+    <!-- Google Tag Manager (noscript) -->
+    <noscript><iframe src="https://www.googletagmanager.com/ns.html?id={{gtmContainerId}}" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
```

Comment:
Tiny point, but we don't need `visibility:hidden` if there's already `display:none`

Reply:
This was the code Google provided so I'm wary of messing with it!

Reply:
Yeah, whoever wrote it doesn't understand CSS!